### PR TITLE
fix(l10n): move l10n install into postinstall

### PR DESCRIPTION
**Pull request by vladikoff**
@vbudhram r?

### CODE REVIEW

**Comment by vladikoff** [comment]
Related: https://github.com/mozilla/fxa-auth-mailer/pull/238

File: locale/README.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ This directory contains translated strings for all supported locales. Strings ar
...
 ## How to use
 
-This folder can be used locally, but the default behavior is to update through `bower.json`.  If you do not follow those steps you might lose your work without understanding.
+This folder can be used locally, but the default behavior is to update through `npm install` (see the postinstall `download_l10n.sh` step).  If you do not follow those steps you might lose your work without understanding.
 
 Here are a few things to know:
 
 * Do `git add` the file you are working on as often as possible. Otherwise you might just run `grunt` and it’ll overwrite all your work.
-* The files that the default behavior uses to generate the content is from `app/bower_components/fxa-content-server-l10n`, that is handled by bower.
+* The files that the default behavior uses to generate the content is from `node_modules/fxa-content-server-l10n`, that is handled by bower.
```
Comment (vbudhram) [inline locale/README.md]: Should this be `handled by npm` instead?

Comment (vladikoff) [inline locale/README.md]: correct, thanks

**Comment by shane-tomlinson** [comment]
Is the purpose of this PR to allow pinning to a particular git SHA via the environment variable? I can't really tell from the commit message.

**Pull request merged**